# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# András Bögöly <ebogoly@gmail.com>, 2011,2014
# Peter Borsa <peter.borsa@gmail.com>, 2014
# teknos.ferenc <teknos.ferenc@gmail.com>, 2014
# Zoltan Hoppár <zoltanh721@fedoraproject.org>, 2014
msgid ""
msgstr ""
"Project-Id-Version: dnf\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2014-08-06 15:47+0200\n"
"PO-Revision-Date: 2014-08-06 13:47+0000\n"
"Last-Translator: Jan silhan <jsilhan@redhat.com>\n"
"Language-Team: Hungarian (http://www.transifex.com/projects/p/dnf/language/hu/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: hu\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../dnf/persistor.py:235
msgid "Invalid groups database, clearing."
msgstr "Hibás csoportadatbázis, tisztítás alatt."

#: ../dnf/persistor.py:254
msgid "Unsupported installed groups database found, resetting."
msgstr "Nem támogatott csoportadatbázis található, helyrerakás alatt."

#: ../dnf/persistor.py:326
#, python-format
msgid "%s is empty file"
msgstr "%s egy üres fájl"

#: ../dnf/drpm.py:63 ../dnf/repo.py:340
#, python-format
msgid "unsupported checksum type: %s"
msgstr "nem támogatott ellenőrzőösszeg típus: %s"

#: ../dnf/drpm.py:135
msgid "Delta RPM rebuild failed"
msgstr "Delta RPM újraépítés meghiúsult"

#: ../dnf/drpm.py:137
msgid "Checksum of the delta-rebuilt RPM failed"
msgstr "Az ellenőrzőösszeg a delta-újraépített RPM-en meghiúsult"

#: ../dnf/comps.py:348
#, python-format
msgid "Environment '%s' is already installed."
msgstr "A(z) '%s' környezet már telepítve van."

#: ../dnf/comps.py:368 ../dnf/comps.py:390
#, python-format
msgid "Environment '%s' is not installed."
msgstr "A(z) '%s' nevű környezet nincs telepítve."

#: ../dnf/comps.py:415
#, python-format
msgid "Group '%s' is already installed."
msgstr "A(z) '%s' nevű csoport már telepítve van."

#: ../dnf/comps.py:430 ../dnf/comps.py:445
#, python-format
msgid "Group '%s' not installed."
msgstr "A(z) '%s' nevű csoport nincs telepítve."

#: ../dnf/repo.py:673
#, python-format
msgid "Failed to synchronize cache for repo '%s' from '%s': %s"
msgstr "A(z) '%s' gyorsítótár szinkronizációja meghiúsult a következőnél '%s': %s"

#: ../dnf/conf/read.py:52
#, python-format
msgid "Warning: failed loading '%s', skipping."
msgstr "Figyelmeztetés: nem sikerült betölteni: '%s', átugrás."

#: ../dnf/conf/read.py:62
#, python-format
msgid "Repository %r: Error parsing config: %s"
msgstr "%r tároló: Hiba a konfigurációs fájl elemzése során: %s"

#: ../dnf/conf/read.py:68
#, python-format
msgid "Repository %r is missing name in configuration, using id."
msgstr "A(z) %r tárolónak hiányzik a neve a konfigurációból, id használata helyette."

#: ../dnf/conf/__init__.py:74
#, python-format
msgid "Could not set cachedir: %s"
msgstr "Gyorsítótár-mappa beállítása meghiúsult: %s"

#: ../dnf/yum/rpmtrans.py:111
msgid "Cleanup"
msgstr "Tisztítás"

#: ../dnf/yum/rpmtrans.py:112 ../dnf/cli/output.py:946
msgid "Downgrading"
msgstr "Visszaállítás"

#: ../dnf/yum/rpmtrans.py:113
msgid "Erasing"
msgstr "Törlés"

#: ../dnf/yum/rpmtrans.py:114 ../dnf/cli/output.py:942
msgid "Installing"
msgstr "Telepítés"

#: ../dnf/yum/rpmtrans.py:115 ../dnf/cli/output.py:1651
msgid "Obsoleting"
msgstr "Elavulttá tevés"

#: ../dnf/yum/rpmtrans.py:116 ../dnf/cli/output.py:944
msgid "Reinstalling"
msgstr "Újratelepítés"

#: ../dnf/yum/rpmtrans.py:117 ../dnf/cli/output.py:943
msgid "Upgrading"
msgstr "Frissítés"

#: ../dnf/yum/history.py:758
#, python-format
msgid "Unable to initialize yumdb history: %s"
msgstr "Nem sikerült a yumdb előzményeinek inicializációja: %s"

#: ../dnf/yum/history.py:1675
#, python-format
msgid "Can not create history database at '%s'."
msgstr "Nem sikerült előzmény-adatbázist készíteni a következőnél: '%s'"

#: ../dnf/cli/option_parser.py:50
#, python-format
msgid "Command line error: %s"
msgstr "Parancssori hiba a következőben: %s"

#: ../dnf/cli/option_parser.py:99
msgid "--color takes one of: auto, always, never"
msgstr "--Szín egyikét veszi fel: auto, mindig, soha"

#: ../dnf/cli/option_parser.py:130
#, python-format
msgid "Options Error: %s"
msgstr "Opció hiba: %s"

#: ../dnf/cli/option_parser.py:141
#, python-format
msgid "--installroot must be an absolute path: %s"
msgstr "A(z) --instalroot -nak abszolút elérési útnak kell lennie: %s"

#: ../dnf/cli/option_parser.py:158
msgid "allow erasing of installed packages to resolve dependencies"
msgstr "engedélyezi a telepített csomagok törlését a függőségek feloldása végett."

#: ../dnf/cli/option_parser.py:161
msgid "try the best available package versions in transactions."
msgstr "megkísérli a legjobb elérhető verziójú csomagokat a tranzakciók alatt."

#: ../dnf/cli/option_parser.py:165
msgid "run entirely from system cache, don't update cache"
msgstr "futtatás a rendszer gyorsítótárából anélkül, hogy a csomagok vagy tárolóinformációk frissítve lennének"

#: ../dnf/cli/option_parser.py:169
msgid "config file location"
msgstr "konfigurációs fájl elérésí útja"

#: ../dnf/cli/option_parser.py:172
msgid "debugging output level"
msgstr "a hibakeresés szintje"

#: ../dnf/cli/option_parser.py:175
msgid "dumps detailed solving results into files"
msgstr "kiírja a részletes feloldási útvonalat fájlokba"

#: ../dnf/cli/option_parser.py:178
msgid "show duplicates, in repos, in list/search commands"
msgstr "ismétlődések mutatása a tárolókban és a keresési parancsokban"

#: ../dnf/cli/option_parser.py:181
msgid "error output level"
msgstr "hiba kimeneti szintje"

#: ../dnf/cli/option_parser.py:183
msgid "debugging output level for rpm"
msgstr "rpm hibakereső szint"

#: ../dnf/cli/option_parser.py:186
msgid "quiet operation"
msgstr "csendes működés"

#: ../dnf/cli/option_parser.py:188
msgid "verbose operation"
msgstr "visszajelzés működés közben"

#: ../dnf/cli/option_parser.py:190
msgid "answer yes for all questions"
msgstr "igennel válaszol minden felmerülő kérdésre"

#: ../dnf/cli/option_parser.py:192
msgid "answer no for all questions"
msgstr "minden kérdésre a válasz nemleges"

#: ../dnf/cli/option_parser.py:194
msgid "show DNF version and exit"
msgstr "A DNF verziójának mutatása, majd kilépés"

#: ../dnf/cli/option_parser.py:195
msgid "set install root"
msgstr "telepítési root beállítása"

#: ../dnf/cli/option_parser.py:204
msgid "exclude packages by name or glob"
msgstr "csomagok kizárása név vagy glob alapján"

#: ../dnf/cli/option_parser.py:207
msgid "disable excludes"
msgstr "a tiltás kizárja"

#: ../dnf/cli/option_parser.py:210
msgid "enable obsoletes processing during upgrades"
msgstr "engedélyezi az elévültek feldolgozását frissítés alatt"

#: ../dnf/cli/option_parser.py:212
msgid "disable all plugins"
msgstr "az összes beépülő modul letiltása"

#: ../dnf/cli/option_parser.py:214
msgid "disable gpg signature checking"
msgstr "gpg-aláírás ellenőrzés kihagyása"

#: ../dnf/cli/option_parser.py:217
msgid "disable plugins by name"
msgstr "bővítmények tiltása név szerint"

#: ../dnf/cli/option_parser.py:220
msgid "control whether color is used"
msgstr "szín használatának befolyásolása"

#: ../dnf/cli/option_parser.py:222
msgid "override the value of $releasever in config and repo files"
msgstr "felülírja a $releasever értékét a konfigurációs és tárolóleíró fájlokban"

#: ../dnf/cli/option_parser.py:226
msgid "set arbitrary config and repo options"
msgstr "tetszőleges konfiguráció- és tárolóbeállítások"

#: ../dnf/cli/option_parser.py:230
msgid "resolve to IPv4 addresses only"
msgstr "csak IPv4 címek feloldása"

#: ../dnf/cli/option_parser.py:233
msgid "resolve to IPv6 addresses only"
msgstr "csak IPv6 címek feloldása"

#: ../dnf/cli/option_parser.py:259
msgid "List of Main Commands"
msgstr "Alapvető parancsok listázása"

#: ../dnf/cli/option_parser.py:260
msgid "List of Plugin Commands"
msgstr "Beépülő modulok parancsainak listázása"

#: ../dnf/cli/utils.py:108
msgid "Running"
msgstr "Fut"

#: ../dnf/cli/utils.py:109
msgid "Sleeping"
msgstr "Alszik"

#: ../dnf/cli/utils.py:110
msgid "Uninterruptible"
msgstr "Nem megszakítható"

#: ../dnf/cli/utils.py:111
msgid "Zombie"
msgstr "Zombi"

#: ../dnf/cli/utils.py:112
msgid "Traced/Stopped"
msgstr "Nyomozott/Megállított"

#: ../dnf/cli/utils.py:113
msgid "Unknown"
msgstr "Ismeretlen"

#: ../dnf/cli/utils.py:133
#, python-format
msgid "  The application with PID %d is: %s"
msgstr "Az alkalmazás a(z) %d PID számmal: %s"

#: ../dnf/cli/utils.py:140
#, python-format
msgid "    Memory : %5s RSS (%5sB VSZ)"
msgstr "    Memória : %5s RSS (%5sB VSZ)"

#: ../dnf/cli/utils.py:145
#, python-format
msgid "    Started: %s - %s ago"
msgstr "    Elindítva: %s - %s"

#: ../dnf/cli/utils.py:147
#, python-format
msgid "    State  : %s"
msgstr "Állapot : %s"

#: ../dnf/cli/cli.py:118
#, python-format
msgid "  Installed: %s-%s at %s"
msgstr "  Telepítve : %s-%s at %s"

#: ../dnf/cli/cli.py:120
#, python-format
msgid "  Built    : %s at %s"
msgstr "  Létrehozva: %s at %s"

#: ../dnf/cli/cli.py:157
msgid "Disk Requirements:\n"
msgstr "Szükséges hely:\n"

#: ../dnf/cli/cli.py:164
msgid ""
"Error Summary\n"
"-------------\n"
msgstr "Hiba összegzés\n-------------\n"

#: ../dnf/cli/cli.py:216
msgid "Operation aborted."
msgstr ""

#: ../dnf/cli/cli.py:218 ../dnf/cli/cli.py:380
#: ../dnf/cli/commands/install.py:110 ../dnf/cli/commands/reinstall.py:101
#: ../dnf/cli/commands/__init__.py:539 ../dnf/cli/commands/__init__.py:629
#: ../dnf/cli/commands/__init__.py:696 ../dnf/cli/commands/__init__.py:742
#: ../dnf/cli/commands/__init__.py:801
msgid "Nothing to do."
msgstr "Nincs tennivaló."

#: ../dnf/cli/cli.py:223
msgid "Downloading Packages:"
msgstr "Csomagok letöltése:"

#: ../dnf/cli/cli.py:230
#, python-format
msgid ""
"Error downloading packages:\n"
"%s"
msgstr "Hiba a csomagok letöltésekor:\n%s"

#: ../dnf/cli/cli.py:241
msgid "Complete!"
msgstr "Kész!"

#: ../dnf/cli/cli.py:262
msgid ""
"Refusing to automatically import keys when running unattended.\n"
"Use \"-y\" to override."
msgstr "Felügyelet nélküli futás közben kulcsok nem importálhatóak.\nHasználjuk az \"-y\" kapcsolót a felülbíráláshoz."

#: ../dnf/cli/cli.py:302 ../dnf/cli/cli.py:455 ../dnf/cli/cli.py:461
msgid "Obsoleting Packages"
msgstr "Elavult csomagok"

#: ../dnf/cli/cli.py:317 ../dnf/cli/commands/group.py:390
#: ../dnf/cli/commands/upgrade.py:91 ../dnf/cli/commands/__init__.py:897
msgid "No packages marked for upgrade."
msgstr "Nincsenek frissítésre kijelölt csomagok"

#: ../dnf/cli/cli.py:339
msgid "No packages marked for distribution synchronization."
msgstr "Nincsenek disztribúció-szinkronizációra kijelölt csomagok"

#: ../dnf/cli/cli.py:370 ../dnf/cli/cli.py:664 ../dnf/cli/cli.py:693
#: ../dnf/cli/commands/install.py:103 ../dnf/cli/commands/__init__.py:531
#, python-format
msgid "No package %s%s%s available."
msgstr "Nincs ilyen csomag: %s%s%s."

#: ../dnf/cli/cli.py:375
#, python-format
msgid "No match for available package: %s"
msgstr "Nem található egyezés a következő csomaghoz: %s."

#: ../dnf/cli/cli.py:431
msgid "Installed Packages"
msgstr "Telepített csomagok"

#: ../dnf/cli/cli.py:439
msgid "Available Packages"
msgstr "Elérhető csomagok"

#: ../dnf/cli/cli.py:443
msgid "Extra Packages"
msgstr "Extra csomagok"

#: ../dnf/cli/cli.py:447
msgid "Upgraded Packages"
msgstr "Frissített csomagok"

#: ../dnf/cli/cli.py:463
msgid "Recently Added Packages"
msgstr "Nemrégiben hozzáadott csomagok"

#: ../dnf/cli/cli.py:467
msgid "No matching Packages to list"
msgstr "Nem található csomag"

#: ../dnf/cli/cli.py:543
msgid "No Matches found"
msgstr "Nincsenek találatok"

#: ../dnf/cli/cli.py:584 ../dnf/cli/commands/__init__.py:1133
#: ../dnf/cli/commands/__init__.py:1149
msgid "No transaction ID given"
msgstr "Nem lett megadva tranzakció azonosító"

#: ../dnf/cli/cli.py:593 ../dnf/cli/cli.py:601 ../dnf/cli/output.py:1797
msgid "Bad transaction ID given"
msgstr "Rossz tranzakció azonosító lett megadva"

#: ../dnf/cli/cli.py:609
msgid "Not found given transaction ID"
msgstr "Nem található a megadott tranzakció azonosítója"

#: ../dnf/cli/cli.py:618 ../dnf/cli/commands/__init__.py:1135
#: ../dnf/cli/commands/__init__.py:1151
msgid "Found more than one transaction ID!"
msgstr "Több, mint egy tranzakció azonosító található!"

#: ../dnf/cli/cli.py:635
#, python-format
msgid "Transaction history is incomplete, before %u."
msgstr "A tranzakciós előzmények hiányosak a következő előtt: %u."

#: ../dnf/cli/cli.py:637
#, python-format
msgid "Transaction history is incomplete, after %u."
msgstr "A tranzakciós előzmények hiányosak a következő után: %u."

#: ../dnf/cli/cli.py:660 ../dnf/cli/cli.py:689
#, python-format
msgid "No package %s%s%s installed."
msgstr "Nem található telepített csomag a következő névvel: %s%s%s."

#: ../dnf/cli/cli.py:750
#, python-format
msgid "Unknown repo: '%s'"
msgstr "Ismeretlen tároló: '%s'"

#: ../dnf/cli/cli.py:818
msgid "You need to give some command"
msgstr "Egy parancs megadása szükségeltetik"

#: ../dnf/cli/cli.py:825
#, python-format
msgid "No such command: %s. Please use %s --help"
msgstr "Nincs ilyen parancs: %s. Kérjük használja a következőt: %s --help"

#: ../dnf/cli/cli.py:927 ../dnf/cli/cli.py:931
#, python-format
msgid "Config error: %s"
msgstr "Konfigurációs hiba: %s."

#: ../dnf/cli/cli.py:1015
msgid "releasever not given and can not be detected from the installroot."
msgstr "A releasever nincs megadva, és nem lehet kideríteni az installroot-ból."

#: ../dnf/cli/cli.py:1045
#, python-format
msgid "Command \"%s\" already defined"
msgstr "\"%s\" nevű parancs már létezik"

#: ../dnf/cli/main.py:77
#, python-format
msgid "Error: %s"
msgstr "Hiba: %s"

#: ../dnf/cli/main.py:97
msgid "Terminated."
msgstr "Megszakítva."

#: ../dnf/cli/main.py:128
msgid "No read/execute access in current directory, moving to /"
msgstr "Nincs olvasási/futtatási jogosultság a jelenlegi könyvtárban, ugrás ide: /"

#: ../dnf/cli/main.py:154
msgid "Dependencies resolved."
msgstr "Függőségek feloldva."

#: ../dnf/cli/output.py:462
msgid "Name"
msgstr "Név"

#: ../dnf/cli/output.py:463 ../dnf/cli/output.py:969
msgid "Arch"
msgstr "Arch"

#: ../dnf/cli/output.py:465
msgid "Epoch"
msgstr "Epocha"

#: ../dnf/cli/output.py:466 ../dnf/cli/output.py:970
msgid "Version"
msgstr "Verzió"

#: ../dnf/cli/output.py:467
msgid "Release"
msgstr "Kiadás"

#: ../dnf/cli/output.py:468 ../dnf/cli/output.py:971
msgid "Size"
msgstr "Méret"

#: ../dnf/cli/output.py:469
msgid "Repo"
msgstr "Tároló"

#: ../dnf/cli/output.py:471
msgid "From repo"
msgstr "Tárolóból"

#: ../dnf/cli/output.py:476
msgid "Buildtime"
msgstr "Build idő"

#: ../dnf/cli/output.py:478
msgid "Install time"
msgstr "Telepítés ideje"

#: ../dnf/cli/output.py:486
msgid "Installed by"
msgstr "Telepítette"

#: ../dnf/cli/output.py:493
msgid "Changed by"
msgstr "Megváltoztatta"

#: ../dnf/cli/output.py:494
msgid "Summary"
msgstr "Összegzés"

#: ../dnf/cli/output.py:496
msgid "URL"
msgstr "URL"

#: ../dnf/cli/output.py:497
msgid "License"
msgstr "Licenc"

#: ../dnf/cli/output.py:498
msgid "Description"
msgstr "Leírás"

#: ../dnf/cli/output.py:618
msgid "y"
msgstr "y"

#: ../dnf/cli/output.py:618
msgid "yes"
msgstr "igen"

#: ../dnf/cli/output.py:619
msgid "n"
msgstr "n"

#: ../dnf/cli/output.py:619
msgid "no"
msgstr "nem"

#: ../dnf/cli/output.py:622
msgid "Is this ok [y/N]: "
msgstr "Ez így jó? [y/N]"

#: ../dnf/cli/output.py:625
msgid "Is this ok [Y/n]: "
msgstr "Ez így jó? [Y/n]: "

#: ../dnf/cli/output.py:701
#, python-format
msgid ""
"\n"
"Group: %s"
msgstr "\nCsoport: %s"

#: ../dnf/cli/output.py:705
#, python-format
msgid " Group-Id: %s"
msgstr " Csoport azonosító: %s"

#: ../dnf/cli/output.py:707
#, python-format
msgid " Description: %s"
msgstr " Leírás: %s"

#: ../dnf/cli/output.py:709
#, python-format
msgid " Language: %s"
msgstr " Nyelv: %s"

#: ../dnf/cli/output.py:712
msgid " Mandatory Packages:"
msgstr " Szükséges csomagok:"

#: ../dnf/cli/output.py:713
msgid " Default Packages:"
msgstr " Alapértelmezett csomagok:"

#: ../dnf/cli/output.py:714
msgid " Optional Packages:"
msgstr " Opcionális csomagok:"

#: ../dnf/cli/output.py:715
msgid " Conditional Packages:"
msgstr " Feltételes csomagok:"

#: ../dnf/cli/output.py:761
#, python-format
msgid "Repo        : %s"
msgstr "Tároló      : %s"

#: ../dnf/cli/output.py:768
msgid "Matched from:"
msgstr "Találat a következőtől:"

#: ../dnf/cli/output.py:772
msgid "Description : "
msgstr "Leírás      : "

#: ../dnf/cli/output.py:775
#, python-format
msgid "URL         : %s"
msgstr "URL         : %s"

#: ../dnf/cli/output.py:778
#, python-format
msgid "License     : %s"
msgstr "Licenc      : %s"

#: ../dnf/cli/output.py:781
#, python-format
msgid "Filename    : %s"
msgstr "Fájlnév     : %s"

#: ../dnf/cli/output.py:785
msgid "Other       : "
msgstr "Egyéb       : "

#: ../dnf/cli/output.py:841
msgid "There was an error calculating total download size"
msgstr "Hiba történt a teljes letöltési méret számítása során"

#: ../dnf/cli/output.py:847
#, python-format
msgid "Total size: %s"
msgstr "Teljes méret: %s"

#: ../dnf/cli/output.py:850
#, python-format
msgid "Total download size: %s"
msgstr "Teljes letöltési méret: %s"

#: ../dnf/cli/output.py:853 ../dnf/cli/output.py:875
#, python-format
msgid "Installed size: %s"
msgstr "Telepített méret: %s"

#: ../dnf/cli/output.py:871
msgid "There was an error calculating installed size"
msgstr "Hiba történt a telepített méret számítása során"

#: ../dnf/cli/output.py:884
msgid "Marking installed:"
msgstr ""

#: ../dnf/cli/output.py:891
msgid "Marking removed:"
msgstr ""

#: ../dnf/cli/output.py:901
msgid "Group"
msgstr ""

#: ../dnf/cli/output.py:901
msgid "Packages"
msgstr ""

#: ../dnf/cli/output.py:945
msgid "Removing"
msgstr "Eltávolítás"

#: ../dnf/cli/output.py:969 ../dnf/cli/output.py:1852
msgid "Package"
msgstr "Csomag"

#: ../dnf/cli/output.py:970
msgid "Repository"
msgstr "Tároló"

#: ../dnf/cli/output.py:983
#, python-format
msgid "     replacing  %s%s%s.%s %s\n"
msgstr "     lecserélés  %s%s%s.%s %s\n"

#: ../dnf/cli/output.py:991
#, python-format
msgid ""
"\n"
"Transaction Summary\n"
"%s\n"
msgstr "\nTranzakció Összegzés\n%s\n"

#: ../dnf/cli/output.py:996 ../dnf/cli/output.py:1647
#: ../dnf/cli/output.py:1648
msgid "Install"
msgstr "Telepítés"

#: ../dnf/cli/output.py:997 ../dnf/cli/output.py:1656
msgid "Upgrade"
msgstr "Frissítés"

#: ../dnf/cli/output.py:998
msgid "Remove"
msgstr "Eltávolítás"

#: ../dnf/cli/output.py:999 ../dnf/cli/output.py:1654
msgid "Downgrade"
msgstr "Visszaállítás"

#: ../dnf/cli/output.py:1085
msgid "Reinstalled"
msgstr "Újratelepítve"

#: ../dnf/cli/output.py:1086
msgid "Removed"
msgstr "Eltávolítva"

#: ../dnf/cli/output.py:1087 ../dnf/cli/output.py:1492
#: ../dnf/cli/output.py:1494
msgid "Installed"
msgstr "Telepítve"

#: ../dnf/cli/output.py:1088 ../dnf/cli/output.py:1493
#: ../dnf/cli/output.py:1657
msgid "Upgraded"
msgstr "Frissítve"

#: ../dnf/cli/output.py:1089 ../dnf/cli/output.py:1493
#: ../dnf/cli/output.py:1655
msgid "Downgraded"
msgstr "Visszaállítva"

#: ../dnf/cli/output.py:1145
msgid "Total"
msgstr "Összesen"

#: ../dnf/cli/output.py:1166
msgid "I"
msgstr "I"

#: ../dnf/cli/output.py:1167
msgid "O"
msgstr "O"

#: ../dnf/cli/output.py:1168
msgid "E"
msgstr "E"

#: ../dnf/cli/output.py:1169
msgid "R"
msgstr "R"

#: ../dnf/cli/output.py:1170
msgid "D"
msgstr "D"

#: ../dnf/cli/output.py:1171
msgid "U"
msgstr "U"

#: ../dnf/cli/output.py:1185
msgid "<unset>"
msgstr "<üres>"

#: ../dnf/cli/output.py:1186
msgid "System"
msgstr "Rendszer"

#: ../dnf/cli/output.py:1267 ../dnf/cli/output.py:1398
msgid "No transactions"
msgstr "Nincsenek tranzakciók"

#: ../dnf/cli/output.py:1292 ../dnf/cli/output.py:1841
#: ../dnf/cli/output.py:1918
msgid "Bad transaction IDs, or package(s), given"
msgstr "Rossz tranzakció azonosító vagy csomagnév lett megadva"

#: ../dnf/cli/output.py:1339
msgid "Command line"
msgstr "Parancssor"

#: ../dnf/cli/output.py:1341 ../dnf/cli/output.py:1729
msgid "Login user"
msgstr "Felhasználó"

#: ../dnf/cli/output.py:1342 ../dnf/cli/output.py:1850
msgid "ID"
msgstr "azonosító"

#: ../dnf/cli/output.py:1344
msgid "Date and time"
msgstr "Dátum és idő"

#: ../dnf/cli/output.py:1345 ../dnf/cli/output.py:1731
#: ../dnf/cli/output.py:1851
msgid "Action(s)"
msgstr "Művelet(ek)"

#: ../dnf/cli/output.py:1346 ../dnf/cli/output.py:1732
msgid "Altered"
msgstr "Változtatta"

#: ../dnf/cli/output.py:1431 ../dnf/cli/output.py:1805
msgid "No transaction ID, or package, given"
msgstr "Nem lett megadva tranzakció azonosító vagy csomag"

#: ../dnf/cli/output.py:1492
msgid "Erased"
msgstr "Törölve"

#: ../dnf/cli/output.py:1494
msgid "Not installed"
msgstr "Nem telepítve"

#: ../dnf/cli/output.py:1495
msgid "Older"
msgstr "Régebbi"

#: ../dnf/cli/output.py:1495
msgid "Newer"
msgstr "Újabb"

#: ../dnf/cli/output.py:1532 ../dnf/cli/output.py:1534
#: ../dnf/cli/output.py:1939
msgid "Transaction ID :"
msgstr "Tranzakció azonosító:"

#: ../dnf/cli/output.py:1536 ../dnf/cli/output.py:1941
msgid "Begin time     :"
msgstr "Kezdés ideje   :"

#: ../dnf/cli/output.py:1539 ../dnf/cli/output.py:1541
msgid "Begin rpmdb    :"
msgstr "rpmdb kezdete:"

#: ../dnf/cli/output.py:1557
#, python-format
msgid "(%u seconds)"
msgstr "(%u másodperc)"

#: ../dnf/cli/output.py:1559
#, python-format
msgid "(%u minutes)"
msgstr "(%u perc)"

#: ../dnf/cli/output.py:1561
#, python-format
msgid "(%u hours)"
msgstr "(%u óra)"

#: ../dnf/cli/output.py:1563
#, python-format
msgid "(%u days)"
msgstr "(%u nap)"

#: ../dnf/cli/output.py:1564
msgid "End time       :"
msgstr "Befejezés ideje:"

#: ../dnf/cli/output.py:1567 ../dnf/cli/output.py:1569
msgid "End rpmdb      :"
msgstr "rpmdb befejezése:"

#: ../dnf/cli/output.py:1572 ../dnf/cli/output.py:1574
msgid "User           :"
msgstr "Felhasználó    :"

#: ../dnf/cli/output.py:1578 ../dnf/cli/output.py:1581
#: ../dnf/cli/output.py:1583 ../dnf/cli/output.py:1585
#: ../dnf/cli/output.py:1587
msgid "Return-Code    :"
msgstr "Visszatérési érték:"

#: ../dnf/cli/output.py:1578 ../dnf/cli/output.py:1583
msgid "Aborted"
msgstr "Megszakítva"

#: ../dnf/cli/output.py:1581
msgid "Failures:"
msgstr "Hibák:"

#: ../dnf/cli/output.py:1585
msgid "Failure:"
msgstr "Hiba:"

#: ../dnf/cli/output.py:1587
msgid "Success"
msgstr "Siker"

#: ../dnf/cli/output.py:1592 ../dnf/cli/output.py:1594
#: ../dnf/cli/output.py:1970
msgid "Command Line   :"
msgstr "Parancssor     :"

#: ../dnf/cli/output.py:1603
#, python-format
msgid "Additional non-default information stored: %d"
msgstr "További nem alapértelmezett információ lett eltárolva: %d"

#: ../dnf/cli/output.py:1608
msgid "Transaction performed with:"
msgstr "A tranzakció lezajlott a következővel:"

#: ../dnf/cli/output.py:1612
msgid "Packages Altered:"
msgstr "Módosított csomagok:"

#: ../dnf/cli/output.py:1616
msgid "Packages Skipped:"
msgstr "Kihagyott csomagok:"

#: ../dnf/cli/output.py:1624
msgid "Rpmdb Problems:"
msgstr "Rpmdb hibák:"

#: ../dnf/cli/output.py:1635
msgid "Scriptlet output:"
msgstr "Scriptlet kimenet:"

#: ../dnf/cli/output.py:1641
msgid "Errors:"
msgstr "Hibák:"

#: ../dnf/cli/output.py:1649
msgid "Dep-Install"
msgstr "Függőség-Telepítés"

#: ../dnf/cli/output.py:1650
msgid "Obsoleted"
msgstr "Elévült"

#: ../dnf/cli/output.py:1652
msgid "Erase"
msgstr "Törlés"

#: ../dnf/cli/output.py:1653
msgid "Reinstall"
msgstr "Újratelepítés"

#: ../dnf/cli/output.py:1730
msgid "Time"
msgstr "Időpont"

#: ../dnf/cli/output.py:1756
msgid "Last day"
msgstr "1 napja"

#: ../dnf/cli/output.py:1757
msgid "Last week"
msgstr "1 hete"

#: ../dnf/cli/output.py:1758
msgid "Last 2 weeks"
msgstr "2 hete"

#: ../dnf/cli/output.py:1759
msgid "Last 3 months"
msgstr "3 hónapja"

#: ../dnf/cli/output.py:1760
msgid "Last 6 months"
msgstr "6 hónapja"

#: ../dnf/cli/output.py:1761
msgid "Last year"
msgstr "1 éve"

#: ../dnf/cli/output.py:1762
msgid "Over a year ago"
msgstr "Több, mint egy éve"

#: ../dnf/cli/output.py:1809
#, python-format
msgid "No Transaction %s found"
msgstr "Nem található a következő tranzakció: %s"

#: ../dnf/cli/output.py:1815
msgid "Transaction ID:"
msgstr "Tranzakció azonosító:"

#: ../dnf/cli/output.py:1816
msgid "Available additional history information:"
msgstr "Rendelkezésre álló további információ az előzményekről:"

#: ../dnf/cli/output.py:1829
#, python-format
msgid "%s: No additional data found by this name"
msgstr "%s: Nem található további adat erre a névre"

#: ../dnf/cli/output.py:1942
msgid "Package        :"
msgstr "Csomag :"

#: ../dnf/cli/output.py:1943
msgid "State          :"
msgstr "Állapot :"

#: ../dnf/cli/output.py:1946
msgid "Size           :"
msgstr "Méret :"

#: ../dnf/cli/output.py:1948
msgid "Build host     :"
msgstr "Build kiszolgáló :"

#: ../dnf/cli/output.py:1951
msgid "Build time     :"
msgstr "Build idő :"

#: ../dnf/cli/output.py:1953
msgid "Packager       :"
msgstr "Csomagoló :"

#: ../dnf/cli/output.py:1955
msgid "Vendor         :"
msgstr "Kiadó :"

#: ../dnf/cli/output.py:1957
msgid "License        :"
msgstr "Licenc :"

#: ../dnf/cli/output.py:1959
msgid "URL            :"
msgstr "URL :"

#: ../dnf/cli/output.py:1961
msgid "Source RPM     :"
msgstr "Forrás RPM :"

#: ../dnf/cli/output.py:1964
msgid "Commit Time    :"
msgstr "Beküldés ideje :"

#: ../dnf/cli/output.py:1966
msgid "Committer      :"
msgstr "Beküldő :"

#: ../dnf/cli/output.py:1968
msgid "Reason         :"
msgstr "Oka :"

#: ../dnf/cli/output.py:1972
msgid "From repo      :"
msgstr "Ebből a tárolóból :"

#: ../dnf/cli/output.py:1976
msgid "Installed by   :"
msgstr "Telepítette :"

#: ../dnf/cli/output.py:1980
msgid "Changed by     :"
msgstr "Megváltoztatta :"

#: ../dnf/cli/output.py:2011
msgid "installed"
msgstr "telepítve"

#: ../dnf/cli/output.py:2012
msgid "an upgrade"
msgstr "egy frissítés"

#: ../dnf/cli/output.py:2013
msgid "erased"
msgstr "törölve"

#: ../dnf/cli/output.py:2014
msgid "reinstalled"
msgstr "újratelepítve"

#: ../dnf/cli/output.py:2015
msgid "a downgrade"
msgstr "egy visszaállítás"

#: ../dnf/cli/output.py:2016
msgid "obsoleting"
msgstr "visszaállítás"

#: ../dnf/cli/output.py:2017
msgid "upgraded"
msgstr "frissítve"

#: ../dnf/cli/output.py:2018
msgid "obsoleted"
msgstr "elavult"

#: ../dnf/cli/output.py:2019
msgid "downgraded"
msgstr "visszaállítva"

#: ../dnf/cli/output.py:2022
#, python-format
msgid "---> Package %s.%s %s will be %s"
msgstr "---> A csomag %s.%s %s a következő lesz %s"

#: ../dnf/cli/output.py:2029
msgid "--> Starting dependency resolution"
msgstr "--> Függőségek feloldásának kezdete"

#: ../dnf/cli/output.py:2034
msgid "--> Finished dependency resolution"
msgstr "--> A függőségek feloldása befejeződött"

#: ../dnf/cli/output.py:2169
msgid "Verifying"
msgstr "Ellenőrzés"

#: ../dnf/cli/commands/install.py:42
msgid "Install a package or packages on your system"
msgstr "Csomag(ok) telepítése a rendszerre"

#: ../dnf/cli/commands/install.py:43 ../dnf/cli/commands/reinstall.py:43
#: ../dnf/cli/commands/upgrade.py:42 ../dnf/cli/commands/distrosync.py:35
#: ../dnf/cli/commands/upgradeto.py:32 ../dnf/cli/commands/downgrade.py:35
#: ../dnf/cli/commands/__init__.py:223 ../dnf/cli/commands/__init__.py:269
#: ../dnf/cli/commands/__init__.py:398
msgid "PACKAGE"
msgstr "CSOMAG"

#: ../dnf/cli/commands/install.py:87
#, python-format
msgid "Warning: Group '%s' does not exist."
msgstr "Figyelmeztetés: A(z) '%s' csoport nem létezik."

#: ../dnf/cli/commands/install.py:92
msgid "No packages in any requested group available to install or upgrade."
msgstr "Nem telepíthető illetve frissíthető egy csomag sem a kívánt csoportokból."

#: ../dnf/cli/commands/reinstall.py:42
msgid "reinstall a package"
msgstr "egy csomag újratelepítése"

#: ../dnf/cli/commands/reinstall.py:85 ../dnf/cli/commands/upgrade.py:86
#: ../dnf/cli/commands/__init__.py:297 ../dnf/cli/commands/__init__.py:611
#: ../dnf/cli/commands/__init__.py:678 ../dnf/cli/commands/__init__.py:793
#: ../dnf/cli/commands/__init__.py:845 ../dnf/cli/commands/__init__.py:891
#, python-format
msgid "No match for argument: %s"
msgstr "Nem található egyezés a következő argumentumra: %s"

#: ../dnf/cli/commands/reinstall.py:91 ../dnf/cli/commands/__init__.py:618
#: ../dnf/cli/commands/__init__.py:685
#, python-format
msgid " (from %s)"
msgstr "(ebből: %s)"

#: ../dnf/cli/commands/reinstall.py:92 ../dnf/cli/commands/__init__.py:619
#: ../dnf/cli/commands/__init__.py:686
#, python-format
msgid "Installed package %s%s%s%s not available."
msgstr "A következő telepített csomag nem elérhető: %s%s%s%s"

#: ../dnf/cli/commands/group.py:45
msgid "Error: Need a group or list of groups"
msgstr "Hiba: Szükség van legalább egy csoportra"

#: ../dnf/cli/commands/group.py:89
#, python-format
msgid "No relevant match for the specified '%s'."
msgstr "Nincs releváns egyezés a megadott adathoz: '%s'."

#: ../dnf/cli/commands/group.py:105
msgid "Display, or use, the groups information"
msgstr "Megjeleníti, vagy alkalmazza, a csoportinformációkat"

#: ../dnf/cli/commands/group.py:106
msgid "GROUP"
msgstr "CSOPORT"

#: ../dnf/cli/commands/group.py:144
msgid "No group data available for configured repositories."
msgstr "Nem érhetők el csoportadatok a konfigurált tárolókhoz."

#: ../dnf/cli/commands/group.py:196
#, python-format
msgid "Warning: Group %s does not exist."
msgstr "Figyelem: Az alábbi csoport nem létezik: %s"

#: ../dnf/cli/commands/group.py:226
#, python-format
msgid "Warning: No groups match: %s"
msgstr "Figyelem: Egy csoport sem azonos a következővel: %s"

#: ../dnf/cli/commands/group.py:249
msgid "Available environment groups:"
msgstr "Elérhető környezeti csoportok:"

#: ../dnf/cli/commands/group.py:250
msgid "Installed environment groups:"
msgstr "Telepített környezeti csoportok:"

#: ../dnf/cli/commands/group.py:256
msgid "Installed groups:"
msgstr "Telepített csoportok:"

#: ../dnf/cli/commands/group.py:263
msgid "Installed language groups:"
msgstr "Telepített nyelvi csoportok:"

#: ../dnf/cli/commands/group.py:270
msgid "Available groups:"
msgstr "Elérhető csoportok:"

#: ../dnf/cli/commands/group.py:277
msgid "Available language groups:"
msgstr "Elérhető nyelvi csoportok:"

#: ../dnf/cli/commands/group.py:293
#, python-format
msgid "Environments marked installed: %s"
msgstr "Környezetek telepítettnek jelölve: %s"

#: ../dnf/cli/commands/group.py:298
#, python-format
msgid "Groups marked installed: %s"
msgstr "Csoportok telepítettnek jelölve: %s"

#: ../dnf/cli/commands/group.py:312
#, python-format
msgid "Environments marked removed: %s"
msgstr "Környezetek eltávolításra jelölve: %s"

#: ../dnf/cli/commands/group.py:317
#, python-format
msgid "Groups marked removed: %s"
msgstr "Csoportok eltávolításra jelölve: %s"

#: ../dnf/cli/commands/group.py:357
msgid "Installed Groups:"
msgstr "Telepített csoportok:"

#: ../dnf/cli/commands/group.py:364
msgid "Installed Language Groups:"
msgstr "Telepített nyelvi csoportok:"

#: ../dnf/cli/commands/group.py:371
msgid "Available Groups:"
msgstr "Elérhető csoportok:"

#: ../dnf/cli/commands/group.py:378
msgid "Available Language Groups:"
msgstr "Elérhető nyelvi csoportok:"

#: ../dnf/cli/commands/group.py:421
#, python-format
msgid "Invalid groups sub-command, use: %s."
msgstr "Érvénytelen csoport alparancs, kérjük használja ezt: %s."

#: ../dnf/cli/commands/upgrade.py:41
msgid "Upgrade a package or packages on your system"
msgstr "Csomagok frissítése az ön rendszerében"

#: ../dnf/cli/commands/distrosync.py:34
msgid "Synchronize installed packages to the latest available versions"
msgstr "Telepített csomagok szinkronizálása a legfrissebb verzióhoz"

#: ../dnf/cli/commands/clean.py:43
#, python-format
msgid "Error: clean requires an option: %s"
msgstr "Hiba: a clean parancsnak szüksége van egy opcióra: %s"

#: ../dnf/cli/commands/clean.py:49
#, python-format
msgid "Error: invalid clean argument: %r"
msgstr "Hiba: helytelen clean argumentum: %r"

#: ../dnf/cli/commands/clean.py:75
#, python-format
msgid "Cannot remove %s file %s"
msgstr "Nem távolítható el a %s nevű fájl: %s"

#: ../dnf/cli/commands/clean.py:80
#, python-format
msgid "%s file %s removed"
msgstr "%s fájl %s eltávolította"

#: ../dnf/cli/commands/clean.py:132
msgid "The enabled repos were expired"
msgstr "Az engedélyezett tárolók lejártak"

#: ../dnf/cli/commands/clean.py:141
msgid "Remove cached data"
msgstr "Gyorsítótárazott adat eltávolítása"

#: ../dnf/cli/commands/clean.py:174
msgid "Cleaning repos: "
msgstr "Tárolók tisztítása"

#: ../dnf/cli/commands/clean.py:181
msgid "Cleaning up Everything"
msgstr "Minden tisztítása"

#: ../dnf/cli/commands/clean.py:194
msgid "Cleaning up Packages"
msgstr "Csomagok tisztítása"

#: ../dnf/cli/commands/clean.py:197
msgid "Cleaning up xml metadata"
msgstr "xml metaadat tisztítása"

#: ../dnf/cli/commands/clean.py:200
msgid "Cleaning up database cache"
msgstr "Adatbázis-gyorsítótár tisztítása"

#: ../dnf/cli/commands/clean.py:203
msgid "Cleaning up expire-cache metadata"
msgstr "Elévülési gyorsítótár tisztítása"

#: ../dnf/cli/commands/clean.py:206
msgid "Cleaning up cached rpmdb data"
msgstr "Gyorsítótárazott rpmdb adatok tisztítása"

#: ../dnf/cli/commands/repolist.py:36
msgid "unknown"
msgstr "ismeretlen"

#: ../dnf/cli/commands/repolist.py:38
#, python-format
msgid "Never (last: %s)"
msgstr "Soha (utoljára: %s)"

#: ../dnf/cli/commands/repolist.py:40
#, python-format
msgid "Instant (last: %s)"
msgstr "Most (utoljára: %s)"

#: ../dnf/cli/commands/repolist.py:43
#, python-format
msgid "%s second(s) (last: %s)"
msgstr "%s másodperc (utoljára: %s)"

#: ../dnf/cli/commands/repolist.py:75
msgid "Display the configured software repositories"
msgstr "Beállított szoftverforrások megjelenítése"

#: ../dnf/cli/commands/repolist.py:116 ../dnf/cli/commands/repolist.py:117
msgid "enabled"
msgstr "engedélyezett"

#: ../dnf/cli/commands/repolist.py:135 ../dnf/cli/commands/repolist.py:136
msgid "disabled"
msgstr "tiltott"

#: ../dnf/cli/commands/repolist.py:151
msgid "Repo-id      : "
msgstr "Tároló-azonosító: "

#: ../dnf/cli/commands/repolist.py:152
msgid "Repo-name    : "
msgstr "Tároló-név   : "

#: ../dnf/cli/commands/repolist.py:155
msgid "Repo-status  : "
msgstr "Tároló-állapot: "

#: ../dnf/cli/commands/repolist.py:158
msgid "Repo-revision: "
msgstr "Tároló-vizsgálat:"

#: ../dnf/cli/commands/repolist.py:162
msgid "Repo-tags    : "
msgstr "Tároló-címkék : "

#: ../dnf/cli/commands/repolist.py:167
msgid "Repo-distro-tags: "
msgstr "Tároló-diszt.-címkék: "

#: ../dnf/cli/commands/repolist.py:173
msgid "Repo-updated : "
msgstr "Tároló-frissítve: "

#: ../dnf/cli/commands/repolist.py:175
msgid "Repo-pkgs    : "
msgstr "Tároló-csomagok: "

#: ../dnf/cli/commands/repolist.py:176
msgid "Repo-size    : "
msgstr "Tároló-méret  : "

#: ../dnf/cli/commands/repolist.py:179
msgid "Repo-metalink: "
msgstr "Tároló-metalink: "

#: ../dnf/cli/commands/repolist.py:183
msgid "  Updated    : "
msgstr "  Frissítve  : "

#: ../dnf/cli/commands/repolist.py:186
msgid "Repo-mirrors : "
msgstr "Tároló-tükrök: "

#: ../dnf/cli/commands/repolist.py:190 ../dnf/cli/commands/repolist.py:194
msgid "Repo-baseurl : "
msgstr "Tároló-baseurl: "

#: ../dnf/cli/commands/repolist.py:197
msgid "Repo-expire  : "
msgstr "Tároló-lejárat: "

#: ../dnf/cli/commands/repolist.py:200
msgid "Repo-exclude : "
msgstr "Tároló-kizárás:"

#: ../dnf/cli/commands/repolist.py:204
msgid "Repo-include : "
msgstr "Tároló-tartalmaz: "

#: ../dnf/cli/commands/repolist.py:208
msgid "Repo-excluded: "
msgstr "Tároló-kizárt: "

#: ../dnf/cli/commands/repolist.py:212
msgid "Repo-filename: "
msgstr "Tároló-fájlnév: "

#: ../dnf/cli/commands/repolist.py:220 ../dnf/cli/commands/repolist.py:249
msgid "repo id"
msgstr "tároló azon"

#: ../dnf/cli/commands/repolist.py:237 ../dnf/cli/commands/repolist.py:238
#: ../dnf/cli/commands/repolist.py:254
msgid "status"
msgstr "állapot"

#: ../dnf/cli/commands/repolist.py:250
msgid "repo name"
msgstr "tároló név"

#: ../dnf/cli/commands/upgradeto.py:31
msgid "Upgrade a package on your system to the specified version"
msgstr "Csomag frissítése a meghatározott verzióra az ön rendszerében"

#: ../dnf/cli/commands/downgrade.py:34
msgid "downgrade a package"
msgstr "egy csomag visszaállítása"

#: ../dnf/cli/commands/search.py:42
msgid "Search package details for the given string"
msgstr "Csomagrészletek keresése egy megadott szöveg alapján"

#: ../dnf/cli/commands/search.py:43
msgid "QUERY_STRING"
msgstr "LEKÉRDEZŐ_SZÖVEG"

#: ../dnf/cli/commands/search.py:65
#, python-format
msgid "N/S Matched: %s"
msgstr "N/S Megegyezett: %s"

#: ../dnf/cli/commands/search.py:67
#, python-format
msgid "Matched: %s"
msgstr "Találat: %s"

#: ../dnf/cli/commands/search.py:84
msgid "No matches found."
msgstr ""

#: ../dnf/cli/commands/search.py:104
msgid "Searching Packages: "
msgstr "Csomagok keresése: "

#: ../dnf/cli/commands/__init__.py:42
#, python-format
msgid "To diagnose the problem, try running: '%s'."
msgstr "A probléma diagnosztizálásához próbálja futtatni a következőt: '%s'."

#: ../dnf/cli/commands/__init__.py:44
#, python-format
msgid "To fix inconsistent RPMDB, try running: '%s'."
msgstr "Az inkonzisztens RPMDB javításához próbálja futtatni a következőt: '%s'."

#: ../dnf/cli/commands/__init__.py:46
#, python-format
msgid "If the above doesn't help please report this error at '%s'."
msgstr "Ha a fenti javaslat nem segít, kérjük jelentse a hibát itt: '%s'."

#: ../dnf/cli/commands/__init__.py:54
msgid " Mini usage:\n"
msgstr " Használat:\n"

#: ../dnf/cli/commands/__init__.py:69
msgid ""
"\n"
"You have enabled checking of packages via GPG keys. This is a good thing.\n"
"However, you do not have any GPG public keys installed. You need to download\n"
"the keys for packages you wish to install and install them.\n"
"You can do that by running the command:\n"
"    rpm --import public.gpg.key\n"
"\n"
"\n"
"Alternatively you can specify the url to the key you would like to use\n"
"for a repository in the 'gpgkey' option in a repository section and yum\n"
"will install it for you.\n"
"\n"
"For more information contact your distribution or package provider.\n"
msgstr "\nÖn engedélyezte a csomagok GPG kulcs alapú ellenőrzését. Ez egy jó dolog. Viszont nem található egyetlen telepített GPG kulcs sem, ezért le kéne tölteni a csomagokhoz tartozó kulcsokat, és telepíteni őket. A következő parancs segítségével végezheti ezt el:\nrpm --import public.gpg.key\n\n\nAkár megadhatja a használni kívánt url címét is kulcsnak a tárolóknál a 'gpgkey' opció alatt, így a yum telepíteni fogja ezt önnek.\n\nTovábbi információért lépjen kapcsolatba a disztribúció illetve csomagszolgáltatójával.\n"

#: ../dnf/cli/commands/__init__.py:84
#, python-format
msgid "Problem repository: %s"
msgstr "Hibás tároló: %s"

#: ../dnf/cli/commands/__init__.py:98
#, python-format
msgid "Error: Need to pass a list of pkgs to %s"
msgstr "Hiba: Meg kell adni a csomagok listáját a következőhöz: %s"

#: ../dnf/cli/commands/__init__.py:114
msgid "Error: Need an item to match"
msgstr "Hiba: egy elemnek legalább egyeznie kell"

#: ../dnf/cli/commands/__init__.py:133
msgid "There are no enabled repos."
msgstr "Nincsenek engedélyezett tárolók"

#: ../dnf/cli/commands/__init__.py:222
msgid "Display details about a package or group of packages"
msgstr "Részletek megjelenítése egy csomagról vagy egy csomagcsoportról"

#: ../dnf/cli/commands/__init__.py:252
msgid "List a package or groups of packages"
msgstr "Csomag vagy csomagcsoport listázása"

#: ../dnf/cli/commands/__init__.py:268
msgid "Remove a package or packages from your system"
msgstr "Csomag(ok) eltávolítása a rendszerből"

#: ../dnf/cli/commands/__init__.py:303 ../dnf/cli/commands/__init__.py:851
msgid "No packages marked for removal."
msgstr "Nincsenek eltávolításra kijelölt csomagok."

#: ../dnf/cli/commands/__init__.py:311
msgid "Generate the metadata cache"
msgstr "Metaadat-gyorsítótár generálása"

#: ../dnf/cli/commands/__init__.py:324
msgid "Making cache files for all metadata files."
msgstr "Gyorsítótár készítése az összes metaadat fájlhoz."

#: ../dnf/cli/commands/__init__.py:331
msgid "Metadata timer caching disabled when running on a battery."
msgstr "Metaadat időzítő gyorsítótár nem lesz használatban, ha akkumulátorról működik."

#: ../dnf/cli/commands/__init__.py:336
msgid "Metadata timer caching disabled."
msgstr "A metaadat időzítő gyorsítótár nincs letiltva."

#: ../dnf/cli/commands/__init__.py:341
msgid "Metadata cache refreshed recently."
msgstr "A metaadat gyorsítótár mostanában került frissítésre."

#: ../dnf/cli/commands/__init__.py:364
msgid "Metadata cache created."
msgstr "Metaadat gyorsítótár került létrehozásra."

#: ../dnf/cli/commands/__init__.py:374
msgid "Find what package provides the given value"
msgstr "Megtalálja, mely csomag tartalmazza a megadott értéket"

#: ../dnf/cli/commands/__init__.py:375
msgid "SOME_STRING"
msgstr "EGY_SZÖVEG"

#: ../dnf/cli/commands/__init__.py:397
msgid "Check for available package upgrades"
msgstr "Elérhető csomagfrissítések keresése"

#: ../dnf/cli/commands/__init__.py:522 ../dnf/cli/commands/__init__.py:600
#: ../dnf/cli/commands/__init__.py:667
msgid "No package available."
msgstr "Nincs elérhető csomag."

#: ../dnf/cli/commands/__init__.py:598
msgid "No package installed."
msgstr "Nincs telepített csomag."

#: ../dnf/cli/commands/__init__.py:664 ../dnf/cli/commands/__init__.py:779
#: ../dnf/cli/commands/__init__.py:835
msgid "No package installed from the repository."
msgstr "Nincs telepített csomag a tárolóból."

#: ../dnf/cli/commands/__init__.py:918
msgid "Error: Requires at least one package specification"
msgstr "Hiba: Legalább egy csomagspecifikáció szükségeltetik"

#: ../dnf/cli/commands/__init__.py:941
msgid "Run commands on top of all packages in given repository"
msgstr "Egy meghatározott tároló feletti összes csomagon futtatja az adott parancsokat"

#: ../dnf/cli/commands/__init__.py:944
msgid "REPO"
msgstr "TÁROLÓ"

#: ../dnf/cli/commands/__init__.py:944
msgid "ARG"
msgstr "ARG"

#: ../dnf/cli/commands/__init__.py:989
msgid "Error: Requires a repo ID and a valid sub-command"
msgstr "Hiba: tárolóazonosító és helyes alparancs szükségeltetik"

#: ../dnf/cli/commands/__init__.py:1016
msgid "Display a helpful usage message"
msgstr "Egy használati tipp mutatása"

#: ../dnf/cli/commands/__init__.py:1017
msgid "COMMAND"
msgstr "PARANCS"

#: ../dnf/cli/commands/__init__.py:1049
#, python-format
msgid "No help available for %s"
msgstr "Nincs súgó az alábbi témakörben: %s"

#: ../dnf/cli/commands/__init__.py:1054
msgid ""
"\n"
"\n"
"aliases: "
msgstr "\n\nalias-ok: "

#: ../dnf/cli/commands/__init__.py:1056
msgid ""
"\n"
"\n"
"alias: "
msgstr "\n\nalias: "

#: ../dnf/cli/commands/__init__.py:1073
msgid "Display, or use, the transaction history"
msgstr "Korábbi tranzakciók használata vagy megjelenítése"

#: ../dnf/cli/commands/__init__.py:1083
#, python-format
msgid ""
"Cannot undo transaction %s, doing so would result in an inconsistent package"
" database."
msgstr "Nem lehet visszavonni a következő tranzakciót: %s. Ennek a következménye inkonzisztens csomagadatbázisban mutatkozna meg."

#: ../dnf/cli/commands/__init__.py:1087
#, python-format
msgid ""
"Cannot rollback transaction %s, doing so would result in an inconsistent "
"package database."
msgstr "Nem lehet visszagörgetni a következő tranzakciót: %s. Ennek a következménye inkonzisztens csomagadatbázisban mutatkozna meg."

#: ../dnf/cli/commands/__init__.py:1169
msgid "Transactions:"
msgstr "Műveletek:"

#: ../dnf/cli/commands/__init__.py:1170
msgid "Begin time  :"
msgstr "Kezdés ideje :"

#: ../dnf/cli/commands/__init__.py:1171
msgid "End time    :"
msgstr "Befejezés ideje :"

#: ../dnf/cli/commands/__init__.py:1172
msgid "Counts      :"
msgstr "Találat :"

#: ../dnf/cli/commands/__init__.py:1173
msgid "  NEVRAC :"
msgstr "  NEVRAC :"

#: ../dnf/cli/commands/__init__.py:1174
msgid "  NEVRA  :"
msgstr "  NEVRA  :"

#: ../dnf/cli/commands/__init__.py:1175
msgid "  NA     :"
msgstr "  NA     :"

#: ../dnf/cli/commands/__init__.py:1176
msgid "  NEVR   :"
msgstr "  NEVR   :"

#: ../dnf/cli/commands/__init__.py:1177
msgid "  rpm DB :"
msgstr "  rpm DB :"

#: ../dnf/cli/commands/__init__.py:1178
msgid "  yum DB :"
msgstr "  yum DB :"

#: ../dnf/cli/commands/__init__.py:1197
#, python-format
msgid "Unrecognized options \"%s\"!"
msgstr "Ismeretlen opciók: \"%s\""

#: ../dnf/cli/commands/__init__.py:1214
#, python-format
msgid "Invalid history sub-command, use: %s."
msgstr "Helytelen alparancs az előzményekhez, használja a következőt: %s"

#: ../dnf/cli/commands/__init__.py:1220
msgid "You don't have access to the history DB."
msgstr "Ön nem tudja elérni az előzmények adatbázisát."

#: ../dnf/base.py:101
#, python-format
msgid "%s, disabling."
msgstr "%s, letiltás."

#: ../dnf/base.py:283
#, python-format
msgid "Invalid tsflag in config file: %s"
msgstr "Hibás tsflag a következő konfigurációs fájlban: %s"

#: ../dnf/base.py:336
#, python-format
msgid "Failed to add groups file for repository: %s - %s"
msgstr "Sikertelen a csoportfájl hozzáadása a következő tárolónál: %s - %s"

#: ../dnf/base.py:432
msgid "--> Finding unneeded leftover dependencies"
msgstr "--> Szükségtelen függőségek keresése"

#: ../dnf/base.py:495
msgid "Running transaction check"
msgstr "Tranzakciós ellenőrzés futtatásra"

#: ../dnf/base.py:498
msgid "Error: transaction check vs depsolve:"
msgstr ""

#: ../dnf/base.py:504
msgid "Transaction check succeeded."
msgstr "Tranzakciós ellenőrzés sikerrel járt."

#: ../dnf/base.py:507
msgid "Running transaction test"
msgstr "Tranzakció teszt futtatása"

#: ../dnf/base.py:519
msgid "Transaction check error:\n"
msgstr "Tranzakciós hiba ellenőrzése:\n"

#: ../dnf/base.py:526
msgid "Transaction test succeeded."
msgstr "A tranzakció teszt sikeres."

#: ../dnf/base.py:541
msgid "Running transaction"
msgstr "Tranzakció futtatása"

#: ../dnf/base.py:608
msgid ""
"Warning: scriptlet or other non-fatal errors occurred during transaction."
msgstr "Figyelem: scriptlet vagy egyéb nem végzetes hiba adódott a tranzakció során."

#: ../dnf/base.py:613
#, python-format
msgid "Failed to obtain the transaction lock (logged in as: %s)."
msgstr "Nem sikerült tranzakciós zárat szerezni (bejelentkezve mint: %s)"

#: ../dnf/base.py:616 ../dnf/base.py:627
msgid "Could not run transaction."
msgstr "Tranzakció futtatása meghiúsult."

#: ../dnf/base.py:624
msgid "Transaction couldn't start:"
msgstr "A tranzakció nem indítható:"

#: ../dnf/base.py:636
#, python-format
msgid "Failed to remove transaction file %s"
msgstr "A következő tranzakció-fájl eltávolítása meghiúsult: %s"

#: ../dnf/base.py:693
#, python-format
msgid "%s was supposed to be installed but is not!"
msgstr "A(z) %s nevű csomagnak már telepítve kellett volna lennie, habár nincs."

#: ../dnf/base.py:753
#, python-format
msgid "%s was supposed to be removed but is not!"
msgstr "A(z) %s nevű csomagok már el kellett volna távolítani, habár ez még nem történt meg."

#: ../dnf/base.py:819
#, python-format
msgid "Delta RPMs reduced %.1f MB of updates to %.1f MB (%d.1%% saved)"
msgstr "A DeltaRPMek lecsökkentették a(z) %.1f MB-tnyi frissítést %.1f MB-ra. (%d.1%% megspórolva)"

#: ../dnf/base.py:867
#, python-format
msgid "Public key for %s is not installed"
msgstr "A publikus kulcs nincs telepítve a következőhöz: %s"

#: ../dnf/base.py:871
#, python-format
msgid "Problem opening package %s"
msgstr "Hiba a következő csomag megnyitásánál: %s"

#: ../dnf/base.py:879
#, python-format
msgid "Public key for %s is not trusted"
msgstr "A publikus kulcs nem megbízható a következőhöz: %s"

#: ../dnf/base.py:883
#, python-format
msgid "Package %s is not signed"
msgstr "A következő csomag nincs aláírva: %s"

#: ../dnf/base.py:917
#, python-format
msgid "Cannot remove %s"
msgstr "Nem távolítható el: %s"

#: ../dnf/base.py:921
#, python-format
msgid "%s removed"
msgstr "%s eltávolítva"

#: ../dnf/base.py:1225
#, python-format
msgid "Package %s is already installed, skipping."
msgstr "A(z) %s nevű csomag már telepítve van, átugrás."

#: ../dnf/base.py:1346
#, python-format
msgid "No package %s installed."
msgstr "Nincs %s nevű csomag telepítve."

#: ../dnf/base.py:1482
#, python-format
msgid "Failed to downgrade: %s"
msgstr "Visszaállítás meghiúsult: %s"

#: ../dnf/base.py:1499
#, python-format
msgid "Failed to upgrade: %s"
msgstr "Frissítés meghiúsult: %s"

#: ../dnf/base.py:1630
#, python-format
msgid "Retrieving key from %s"
msgstr "GPG kulcs beszerzése a következőből: %s"

#: ../dnf/base.py:1640
#, python-format
msgid "GPG key retrieval failed: %s"
msgstr "GPG kulcs beszerzése meghiúsult: %s"

#: ../dnf/base.py:1661
#, python-format
msgid "GPG key signature on key %s does not match CA Key for repo: %s"
msgstr "A(z) %s GPG kulcs aláírása nem egyezik a tároló CA kulcsával: %s"

#: ../dnf/base.py:1664
msgid "GPG key signature verified against CA Key(s)"
msgstr "A GPG kulcs sikeresen ellenőrizve lett a CA kulcsokkal"

#: ../dnf/base.py:1672
#, python-format
msgid "Invalid GPG Key from %s: %s"
msgstr "Hibás GPG kulcs a következőből: %s: %s"

#: ../dnf/base.py:1680
#, python-format
msgid "GPG key parsing failed: key does not have value %s"
msgstr "GPG kulcs elemzés meghiúsult: a kulcs nem tartalmaz értéket: %s"

#: ../dnf/base.py:1697
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" Package    : %s (%s)\n"
" From       : %s"
msgstr "%s kulcs importálása 0x%s:\nFelhasználó: \"%s\"\nUjjlenyomat: %s\nCsomag : %s (%s)\nSzármazás: %s"

#: ../dnf/base.py:1706
#, python-format
msgid ""
"Importing %s key 0x%s:\n"
" Userid     : \"%s\"\n"
" Fingerprint: %s\n"
" From       : %s"
msgstr "%s kulcs importálása 0x%s:\nFelhasználó: \"%s\"\nUjjlenyomat: %s\nSzármazás: %s"

#: ../dnf/base.py:1734
#, python-format
msgid ""
"\n"
"\n"
"\n"
" Failing package is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr "\n\n\nA hibás csomag: %s\nA GPG kulcsok konfigurálva vannak, mint: %s\n"

#: ../dnf/base.py:1748
#, python-format
msgid "GPG key at %s (0x%s) is already installed"
msgstr "A következő GPG kulcs már telepítve van: %s (0x%s)"

#: ../dnf/base.py:1784
#, python-format
msgid "Key import failed (code %d)"
msgstr "A kulcs importálása meghiúsult (hibakód %d)"

#: ../dnf/base.py:1786 ../dnf/base.py:1892
msgid "Key imported successfully"
msgstr "A kulcs importálása sikeres"

#: ../dnf/base.py:1790
msgid "Didn't install any keys"
msgstr "Nem lett telepítve egyetlen kulcs sem"

#: ../dnf/base.py:1793
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct for this package.\n"
"Check that the correct key URLs are configured for this repository."
msgstr "A GPG kulcsok a(z) \"%s\" nevű tárolóhoz már telepítve vannak, de nem jók ehhez a csomaghoz.\nKérjük, ellenőrizze, hogy az URL címek helyesen vannak-e megadva ezen tárolóhoz."

#: ../dnf/base.py:1803
msgid "Import of key(s) didn't help, wrong key(s)?"
msgstr "A kulcsok importálása nem segített, rossz kulcsok?"

#: ../dnf/base.py:1826
msgid "No"
msgstr "Nem"

#: ../dnf/base.py:1828
msgid "Yes"
msgstr "Igen"

#: ../dnf/base.py:1829
#, python-format
msgid ""
"\n"
"\n"
"\n"
" CA Key: %s\n"
" Failing repo is: %s\n"
" GPG Keys are configured as: %s\n"
msgstr "\n\n\nCA kulcs: %s\nA hibás tároló: %s\nA GPG kulcsok konfigurálva vannak, mint: %s\n"

#: ../dnf/base.py:1843
#, python-format
msgid "GPG key at %s (0x%s) is already imported"
msgstr "A következő GPG kulcs már telepítve van: %s (0x%s)"

#: ../dnf/base.py:1890
#, python-format
msgid "Key %s import failed"
msgstr "A(z) %s kulcs importálása meghiúsult"

#: ../dnf/base.py:1907
#, python-format
msgid "Didn't install any keys for repo %s"
msgstr "Nem lett telepítve egyetlen kulcs sem a következő tárolóhoz: %s"

#: ../dnf/base.py:1912
#, python-format
msgid ""
"The GPG keys listed for the \"%s\" repository are already installed but they are not correct.\n"
"Check that the correct key URLs are configured for this repository."
msgstr "A GPG kulcsok a(z) \"%s\" nevű tárolóhoz már telepítve vannak, de nem jók.\nKérjük, ellenőrizze, hogy az URL címek helyesen vannak-e megadva ezen tárolóhoz."

#: ../dnf/transaction.py:204
#, python-format
msgid "Will not install a source rpm package (%s)."
msgstr "Nem fogja telepíteni a forrásrpm csomagokat (%s)."

#: ../dnf/plugin.py:126
#, python-format
msgid "Failed loading plugin: %s"
msgstr "A következő beépülő modul betöltése sikertelen: %s"
